evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -85,4 +95,13 @@ def validate(self, X, y):
                                                     data_check_name=self.name,
                                                     message_code=DataCheckMessageCode.CLASS_IMBALANCE_BELOW_THRESHOLD,
                                                     details={"target_values": below_threshold_values}), messages)
+        sample_counts = fold_counts.where(fold_counts < self.min_samples).dropna()
+        if len(below_threshold) and len(sample_counts):
+            sample_count_values = sample_counts.index.tolist()
+            severe_imbalance = [v for v in sample_count_values if v in below_threshold]
+            warning_msg = "The following labels have severe class imbalance because they fall under {:.0f}% of the target and have less than {} samples: {}"
```

Comment:
Nit-pick: "The following labels in the target have severe class imbalanced because" etc